kube/service_test.go

```diff
@@ -90,6 +90,108 @@ func TestServiceOK(t *testing.T) {
 	testhelpers.IsYAMLSubset(assert, expected, actual)
 }
 
+func TestServiceHelmOKDefaults(t *testing.T) {
+	assert := assert.New(t)
+
+	manifest, role := serviceTestLoadRole(assert, "exposed-ports.yml")
+	if manifest == nil || role == nil {
+		return
+	}
+
+	portDef := role.Run.ExposedPorts[0]
+	if !assert.NotNil(portDef) {
+		return
+	}
+	service, err := NewClusterIPService(role, false, false, ExportSettings{
+		CreateHelmChart: true,
+	})
+	if !assert.NoError(err) {
+		return
+	}
+	require.NotNil(t, service)
+	assert.Equal("{{ if .Values.services.loadbalanced }} LoadBalancer {{ else }} ClusterIP {{ end }}", service.Get("spec", "type").String())
```

Comment:
We shouldn't care about the helm templating (just that it expands to the correct thing on both sides), and that can just be two subtests

Reply:
Hm. Will see how sub-tests work. That might help with some other tests where I have full-blown variants checking different render configs.

Reply:
Removed these checks in commit a58f4b7ba69760cd4ffea92b6302a6d7dccefeb2
The RoundtripNode and check coming after handled them already.